### PR TITLE
Move form validation from HTML files into JS files

### PR DESCRIPTION
**Pull request by BryanTriana**

### CODE REVIEW

File: src/main/webapp/edit-profile-page.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    });
+  });
+}
+
+function addProfileFormValidation() {
+  const forms = document.getElementsByClassName('needs-validation');
+
+  const validation = Array.prototype.filter.call(forms, (form) => {
+    form.addEventListener('submit', function(event) {
+      if (form.checkValidity() === false) {
```
Comment (codewithjess) [inline src/main/webapp/edit-profile-page.js:29]:
```suggestion
      if (!form.checkValidity()) {
```

Comment (BryanTriana) [reply src/main/webapp/edit-profile-page.js]: OK I simplified the boolean in last commit.

**Comment by BryanTriana** [comment]
Yes it goes green if valid and red if invalid

**Comment by honey7100** [comment]
> Yes it goes green if valid and red if invalid

Does it have the icons(check mark and x-sign) as well?

**Comment by BryanTriana** [comment]
Yes exacly like it was in the past

**Pull request merged**